js/battle.js

```diff
@@ -2305,7 +2307,7 @@ var Battle = (function () {
 		if (window.console && console.log) console.log(text);
 	};
 	Battle.prototype.log = function (html, preempt) {
-		var willScroll = (this.logFrameElem.scrollTop() + 60 >= this.logElem.height() + this.logPreemptElem.height() - this.logFrameElem.height());
+		var willScroll = (this.logFrameElem.scrollTop() + 60 >= this.logElem.height() + this.logPreemptElem.height() - this.optionsElem.height() - this.logFrameElem.height());
```

Comment:
In principle we can just remove the ``optionsElem`` term from the RHS, can't we?
I am mildly bothered by the magical number 60, but I am not sure if that's what you mean. Why should it be a larger number anyway?